unpackaged/config/dev/objects/Campaign.object

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we want these buttons added to dev or to qa? I am thinking I'd only want them if/when doing qa but dev happens automagically with flow_dev so we'd have to always remember to not commit these changes right?

Reply:
Good point. I was thinking that since they are new components they would not be retrieved automatically, but I was forgetting that buttons are included in the object files rather than separate.

Reply:
@force2b The dev_org flow deploys config/dev. The qa_org flow deploys config/dev and then also config/qa. The latter is probably the right place for metadata that we want to avoid accidentally retrieving and checking in, but that comes with the tradeoff that it's not deployed for dev orgs.

Reply:
I ran a quick test and it didn't pull down the new buttons but it also doesn't appear to be adding them to the layouts either. I also noticed are retrieve is super dirty in Cumulus. Should we do a PR just to clean up the retrieve, I usually like to do that after a new release anyway since there might be new or changed metadata? Maybe after the api v43 change goes in?

Reply:
Yeah, it's as expected that they don't get added to layouts (would be nice, but it'll take some work). Cleaning up the diff after retrieving sounds like a good separate PR after the api change goes in.

Reply:
Not sure what the status on this is. Does this need to get moved to the QA process or is it fine where it is? I did notice that the QA process does not include the OCR and Opportunity.StageName picklist changes that are included in the dev process.

Reply:
@force2b the QA flow deploys both the dev config and the qa config, so it should include that. Did you find that that is not actually working?

Reply:
No. I was just looking in the folders, but I see your point the `config_qa` flow runs the same tasks as `config_dev` plus the `deploy_qa_config` and some others.  

When I create a scratch org I run `cci flow run dev_org'. I assume that only runs the `config_dev` tasks. Is that correct; or should I be using a different flow after I create a scratch org?

Reply:
That's correct. It's up to you whether to run the `qa_org` flow instead of `dev_org`: if you do run the `qa_org` flow, you will also get the result of running `deploy_qa_config`, but then it's up to you to make sure you don't accidentally retrieve and commit those changes.

The metadata in `config/dev` should only include changes that won't be retrieved because they aren't included in the NPSP package.

Reply:
Thinking about this a little more, as it stands right now the `deploy_dev_config` task is run by both the `config_dev` and `config_qa` flows. Given that, I think we should leave the metadata that does the default npsp config and creates the button overrides in the dev folder so these features get installed into both types of orgs. The new metadata that shouldn't be in the package are well named (at least to me) and we really should be encouraging devs to pay attention to what is being checked into the repo. 

Curious what others think -- @jjbennett @dvenor @njjc @lmeerkatz

Reply:
My thoughts:
I prefer to have a clean retrieve, if it is something we do not want pulled down then I think it should be in the qa config so we intentionally run the task when we are reviewing a pr vs dev'ing a pr.
I also think it's easier on a newer devs not to have to worry about doing cherry picking and checkouts when stuff is being retrieved that shouldn't be committed.